code/datums/elements/food/processable.dm

```diff
@@ -0,0 +1,33 @@
+// If an item has the processable item, it can be processed into another item with a specific tool. This adds generic behavior for those actions to make it easier to set-up generically.
+/datum/element/processable
+	element_flags = ELEMENT_BESPOKE
+	id_arg_index = 2
+	///The atom this creates when the processing recipe is used.
+	var/atom/result_atom
```

Comment:
Again, is this an instantiated entity, a type path, it's not clear from the description.

Reply:
I'd assume the fact that it saying "the atom this creates" implies its a type. But I'll make it more clear.